controllers/configuration_controller.go

```diff
@@ -929,6 +1038,36 @@ func (meta *TFConfigurationMeta) createGitAuthConfigVolume() v1.Volume {
 	return gitAuthSecretVolume
 }
 
+func (meta *TFConfigurationMeta) createTerraformCredentialsConfigVolume() v1.Volume {
```

Comment:
The four (three new added and `createGitAuthConfigVolume`) similar function can be unified. Please implement them with a generator function.

Reply:
fixed as per review comments.